app/export.go

```diff
@@ -410,7 +411,8 @@ func (a *App) BuildPostReactions(postId string) (*[]ReactionImportData, *model.A
 		var user *model.User
 		user, err = a.Srv.Store.User().Get(reaction.UserId)
 		if err != nil {
```

Comment:
Let's constrain this new code path to only when the `err` is provably because the user didn't exist (e.g. check error id?), otherwise we might generate invalid exports due to other errors.

Reply:
Good idea!